.circleci/config.yml

```diff
@@ -12,6 +22,241 @@ jobs:
+            ./rainforest-cli -v --skip-update
+            ./rainforest-cli --skip-update run --run-group 9502
+
+  integration_linux_pass_junit:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Build
+          command: |
+            go build
+      - run:
+          name: Start a run that will pass, with junit
+          command: |
+            ./rainforest-cli -v --skip-update
+            ./rainforest-cli --skip-update run --run-group 9502 --junit-file junit-pass.xml
+
+  integration_linux_fail:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Build
+          command: |
+            go build
+      - run:
+          name: Start a run that will fail
+          command: |
+            set +eo pipefail
+            ./rainforest-cli -v --skip-update
+            ./rainforest-cli --skip-update run --run-group 9503
+            if [[ $? != 0 ]]; then
+              echo "TESTING ::: Got the expected non-zero exit code. ✅"
+              echo 0
+            else
+              echo "TESTING ::: Got UNEXPECTED zero exit code. 🚨"
+              echo 1
+            fi
+
+  integration_linux_fail_junit:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Build
+          command: |
+            go build
+      - run:
+          name: Start a run that will fail, with junit
+          command: |
+            set +eo pipefail
+            ./rainforest-cli -v --skip-update
+            ./rainforest-cli --skip-update run --run-group 9503 --junit-file junit-fail.xml
+            if [[ $? != 0 ]]; then
+              echo "TESTING ::: Got the expected non-zero exit code. ✅"
+              exit 0
+            else
+              echo "TESTING ::: Got UNEXPECTED zero exit code. 🚨"
+              exit 1
+            fi
+
+  integration_windows_pass:
+    <<: *windows
+    steps:
+      - checkout
+      - run:
+          name: "Build, and run rainforest"
+          shell: powershell.exe
+          command: |
+            go build
+            .\rainforest-cli.exe -v --skip-update
+            .\rainforest-cli.exe --skip-update run --run-group 9502
+
+  integration_windows_pass_junit:
+    <<: *windows
+    steps:
+      - checkout
+      - run:
+          name: "Build, and run rainforest with junit output"
+          shell: powershell.exe
+          command: |
+            go build
+            .\rainforest-cli.exe -v --skip-update
+            .\rainforest-cli.exe --skip-update run --run-group 9502 --junit-file junit-fail.xml
+
+  integration_windows_fail:
+    <<: *windows
+    steps:
+      - checkout
+      - run:
+          name: "Build, and run rainforest"
+          shell: powershell.exe
+          command: |
+            go build
+            .\rainforest-cli.exe -v --skip-update
+            .\rainforest-cli.exe --skip-update run --run-group 9503
+            if ($LastExitCode -gt 0) {
+              echo "TESTING ::: Got the expected non-zero exit code."
```

Comment:
Should we expect 1 specifically?

Reply:
great point, this should stop it "working" if we change the cli commands